lib/services/plugins-service.ts

```diff
@@ -52,7 +52,37 @@ export class PluginsService implements IPluginsService {
 				this.$errors.fail("Plugin %s is already installed", pluginName);
 			}
 
-			this.configurePlugin(pluginName).wait();
+			var parts = pluginName.split("@");
+			pluginName = parts[0];
+			var version = parts.length > 1 ? parts[1] : "";
+
+			if(!version) {
+				var toLowerCasePluginName = pluginName.toLowerCase();
+
+				var versions:any[] = [];
+				_.each(this.getAvailablePlugins(), (plugin:IPlugin) => {
+					if (plugin.data.Name.toLowerCase() === toLowerCasePluginName || plugin.data.Identifier.toLowerCase() === toLowerCasePluginName) {
+						var pluginVersionsData = (<IMarketplacePlugin>plugin).pluginVersionsData;
+						_.each(pluginVersionsData.Versions, p => {
+							versions.push({name: p.Version, value: p.Version});
+						});
+					}
+				});
+
+				version = versions[0].value;
+				if (versions.length > 1) {
+					var promptData = [{
+						type: "list",
+						name: "version",
+						message: "Which plugin version would you like to use?",
+						choices: versions
+					}];
+					var answer = this.$prompter.get(promptData).wait();
```

Comment:
The code for managing plugin versions and querying the user is in the same function. Can we split them?